### PR TITLE
Features

### PR DESCRIPTION
**Pull request by marurevi**
## Linters
- Add Linters.yml file
- Add .hintrc file
- Add .stylelintrc.json

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<html lang="en">
+<html lang="en"    >
     <head>
         <meta charset="UTF-8">
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (marurevi) [inline index.html:4]: - [x]  For readability, kindly add two spaces of indentation. Do not use the tab key.

File: styles.css [hunk 2]
```diff
@@ -19,7 +19,7 @@
     height: auto;
    
     
-}
+ }
```
Comment (marurevi) [inline styles.css:22]: - [x]  Please, don't leave empty space. They made your code look unclean.

**Review by marurevi — COMMENTED** [review]
## Changes required  ♻
- Please get sure that linters work.
- Correct the indentation
- **please use [this template](https://github.com/microverseinc/readme-template)** for README file.

**Review by Danieldotcomcoder — APPROVED** [review]
Hello @marurevi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏 

Congratulations! 🎉 🎉 🎉 

comp-2
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**